threescale_api/resources.py

```diff
@@ -652,6 +652,8 @@ def __init__(self, entity_name='system_name', **kwargs):
 class Application(DefaultResource):
     def __init__(self, entity_name='system_name', **kwargs):
         super().__init__(entity_name=entity_name, **kwargs)
+        #: Whether Application.api_client should verify SSL; default: None
+        self.api_client_verify = None
```

Comment:
@mganisin How can I set value for `self.api_client_verify`?
I guess I am missing a method on how to set a value for it.

Reply:
application.api_client_verify = True  # or False

Reply:
Thank you

Reply:
I would also consider creating a method, properties tend to be overlooked or even misused. For example, you can have method that will declare the contract (what are the expected values + documentation).
Also just nitpicking - `api_client_verify` isn't that much descriptive name - what that it verify? also, what are the possible values?

Reply:
I would suggest method like:
```
def set_api_client_ssl_verificaton(self, enabled: bool = False):
    """
    Documentation
    """  
    self._api_client_ssl_verify = enabled
```

Reply:
I understand this, I still think that the explicit `setter` would be better, on the other hand - it is not something that should block this one to be merged.

setter:
```python
@property
def api_client_verify(self) -> bool:
   return self._api_client_verify

@api_client_verify.setter
def api_client_verify(self, verify: bool):
    self._api_client_verify = verify
```
This version explicitly states that verify is boolean and value should be boolean.
But again, I'm not against simple property, but I think that this is more readable